tooltester/src/main/java/io/dockstore/tooltester/runWorkflow/WorkflowRunner.java

```diff
@@ -440,20 +445,45 @@ private void setTaskFamily(final List<String> listOfEcsTasksBeforeNewOneWasAdded
         }
     }
 
-    public void uploadRunInfo() {
-        runMetrics.setExecutionStatus(getExecutionStatus());
+    private void uploadRunInfo() {
+        extendedGa4GhApi.executionMetricsPost(new ExecutionsRequestBody().addRunExecutionsItem(runMetrics), Partner.AGC.name(), getEntryNameForApi(), version, "generated with tooltester ('run-workflows-through-wes' command)");
```

Comment:
Similar as the other comment, this can throw an exception which would exit the program unless you catch it somewhere. I don't think you want to exit either because you have other workflow runs that can have their runs uploaded/save

Reply:
It now logs an error message if there is an error and continues on.